### ISSUE TITLE
Autocomplete on branches, doesn't strip the leading plus added by git branch on worktrees

### ISSUE THREAD

**Issue by tankbob** [dahlbyk/posh-git#837]
### System Details

- posh-git version/path: 0.7.3.1  C:\tools\poshgit\dahlbyk-posh-git-9bda399\src
- PowerShell version: 5.1.19041.610
- git version 2.30.1.windows.1
- OS: Microsoft Windows NT 10.0.19041.0

### Issue Description

autocomplete on branches when a branch is checkout to a Worktree

If I'm working in my master branch and I've checked out my Development branch to another worktree folder, I cannot auto complete on a git merge because git branch --no-color puts a + symbol in front of the branch.

```
  Branch1
+ Development                     
+ Release                         
* master                          
```

The auto complete needs to ignore this leading plus when choosing the branch names.  I cannot do `git merge D<tab>` anymore

**Comment by tankbob** [comment]
Horlicks, looking at the source code you've already fixed this, just not released it to chocolatey yet

**Comment by rkeithhill** [comment]
To simplify maintenance of this PowerShell module, we are publishing only to GItHub and the PowerShellGallery. You can install from the PSGallery using `Install-Module posh-git -AllowPrerelease -Scope CurrentUser -Force -AllowClobber` or download from this project's `Releases` page.  Note: On Windows PowerShell, you may need to upgrade PowerShellGet in order to get support for installing pre-release modules.  That issue will go away as soon as we release `1.0.0` in a month or so.

**Comment by dahlbyk** [comment]
Fixed by #805
